app/assets/stylesheets/palette.scss

```diff
@@ -30,4 +30,5 @@ $gray-light: #fafafa;
 $green-white: #EBF5F1;
 :root {
   --grey-lighter: #ededed;
+  --al-toggle-icon-color: #{$stanford-digital-blue};
```

Comment:
For me locally the collection contents sidebar al-toggle-view-children is getting set to:
` --al-toggle-icon-color: rgba(var(--bs-emphasis-color-rgb), 0.65);`

Reply:
That's odd. Here's what I get:
<img width="536" alt="Screenshot 2024-06-07 at 3 09 30 PM" src="https://github.com/sul-dlss/stanford-arclight/assets/92044/06ac7222-bfae-46f2-a60d-489ec713bb83">

Reply:
I've moved this into sulCollection.scss.  Want to give it another shot?